franmolmedo/react-context-and-portals/readme.md

```diff
@@ -0,0 +1,298 @@
+![Header](images/react.png)
```

Comment:
Add **Bold** and *Italic* along the post.

Reply:
It is already done